wiki/en/en-Choosing-a-Server-Type.md

```diff
@@ -46,17 +46,18 @@ Jamulus --nogui --server
 
 See also [Command Line Options](Command-Line-Options) for other parameters you can set.
 
-### 3. Central
-Configuring your server as a Central Server need only be done under special circumstances (e.g. for online events or musical associations). Most people can ignore this type.
+### 3. Directory
+Configuring your server as a Directory Server need only be done under special circumstances (e.g. for online events or musical associations). Most people can ignore this type.
 
-To view servers listed by a custom central server, musicians must enter the address in their client’s "Custom Central Server" settings field.  They will then see a Connection Setup list which is generated by that central server.
+To view servers listed by a custom directory server, musicians must enter the address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by that directory server.
```

Comment:
Along with changing the docs, we have to update multiple places in application labels as well, right?
```
$ grep -ci central src/res/translation/translation_de_DE.ts 
28
```

(Adding this here as "Custom Central Server" is being changed here)

Reply:
Grows a bit, doesn't it?  But the English should be all there now... if I did it right.

Reply:
OK, so now the ts files are there, too.